homeassistant/components/mqtt/sensor.py

```diff
@@ -207,7 +207,9 @@ async def mqtt_async_added_to_hass(self) -> None:
             # MqttEntity.async_added_to_hass(), then we should not restore state
             and not self._expiration_trigger
         ):
-            expiration_at = last_state.last_changed + timedelta(seconds=expire_after)
+            expiration_at = last_state.last_changed + timedelta(
+                seconds=self._expire_after
+            )
```

Comment:
(code-style)
If you keep a walrus above, then this can be kept on one line:
```suggestion
            expiration_at = last_state.last_changed + timedelta(seconds=expire_after)
```